README.md

```diff
@@ -392,10 +391,10 @@ issuing stock is how companies often choose to do this.
   Later we discuss [several other subtleties](#counting-shares) about how shares are counted.
   - 🚧 What is a good overview on stock splits and share buyback.
     Key resources?
-- If you have stock, a key thing to know is the **percentage ownership** of the entire company
-  that this number represents.
-  To determine the percentage of the company a certain number of shares represents, you
-  divide it by the number of outstanding shares.
+- A key insight on the stock you hold is the **percentage ownership** of the entire company
```

Comment:
Nit: It's a thing or a fact, rather than an insight. And "your particular number of shares" sounds odd; it should be "the number of shares you own". I'll edit slightly.

Reply:
Totally fine with me to change, but an insight is just a deeper way of understanding a fact or set of facts or the relationship b/n a series of facts. Penetrating into the true value of the number you have: an insight. Math just a series of insights. Taking it with me to the grave of poetry.

Reply:
Converting this to a definition since it's used in three places.